bedrock/mozorg/templates/mozorg/home/home-de.html

```diff
@@ -12,7 +12,6 @@
 {% block page_title %}Internet für Menschen, nicht für Profit{% endblock %}
```

Comment:
Same suggestion in this file (unable to highlight it since it's not part of the changes in this PR):
```
  {% elif show_pocket_banner %}
    {% include 'includes/banners/pocket.html' %}
```
For a second I thought maybe I'm just missing something as to why they're still in the files, but all of the blocks I'm mentioning have the deleted `banners/pocket.html` file. So just pointing them out :)

Reply:
good catch - bad code searching on my part missed these